src/utils/getCurrentSprint.ts

```diff
@@ -2,12 +2,14 @@ import { isWithinInterval } from "date-fns";
 
 import { type Sprint } from "@/store/features/sprint/sprintSlice";
 
+// const currentDate = new Date(2024, 5, 15);
+
 export function getCurrentSprint(sprints: Sprint[]) {
   const currentDate = new Date();
   const currentSprint = sprints.find((sprint) =>
     isWithinInterval(currentDate, {
       start: sprint.startDate,
-      end: sprint.endDate,
+      end: currentDate,
```

Comment:
change this back to how it was before. Also in the future, you don't need to do it like this, just put the date in there directly on line 6.